winterfell/webapp/gulpfile.js

```diff
@@ -129,6 +138,6 @@ gulp.task('watch', ['build', 'initBrowserSync'], function () {
   gulp.watch('src/**/*.jade', ['jade']);
```

Comment:
you probably want to run the `lint` task after you recompile javascript files from a refresh. (line 136)